dist/dattobd.spec

```diff
@@ -253,7 +263,12 @@ automatically build and install for each kernel.
 
 
 %prep
-%setup -q
+# %setup -q
+%if ! %{with devmode}
+%autosetup
+%else
+%autosetup -n %{name}
+%endif
```

Comment:
Please switch from `%autosetup` to `%setup -q`, in order to maintain compatibility with SLE 11.

For the devmode `%setup` invocation, please use `%setup -q -n %{name}`.